src/components/custom/ActionsByTimeOfDayCustomTooltip.js

```diff
@@ -54,7 +56,7 @@ const CustomTooltip = ({ active, payload, label }) => {
       <div className={classes.customTooltipDiv}>
         <p>{`${label} (${generateAddedTooltipText(label)})`}</p>
         <p className={classes.customTooltipCount}>
-          {`Count : ${payload[0].value}`}
+          {`${t('Count')} : ${payload[0].value}`}
```

Comment:
I think you can bring the `:` inside the `t` function. In some languages (French) you have a space before the colon and in others (English) you don't.

Reply:
Aaah interesting. I did not know this. Recharts in its default Tooltips adds a space, which I didn't like. I'll do this the correct way (for English) by bringing the colon into the `t()`.